### DISCUSSION TITLE
Github bot didn't auto merge the pull request

### DISCUSSION THREAD

**Discussion by ThuWangzw** [lowlighter/metrics#706] [Q&A] [answered]
I follow the steps in README.md, and all goes well except the bot didn't auto-merge the pull request. I need to merge the pull request manually. Could anyone help me? 🙏 🙏 🙏  
![image](https://user-images.githubusercontent.com/26575579/145660127-8a78ae30-747a-4dd7-a0ba-a2dbc71d31b0.png)

**Comment by spenserblack** [comment, marked as answer]
Your output option seems to just be [`pull-request`](https://github.com/ThuWangzw/ThuWangzw/blob/dd5afc9a100689dbb2cf56b4b781969fca004cc6/.github/workflows/action.yml#L28).
Have you tried one of these output options instead?
https://github.com/lowlighter/metrics/blob/cf860cd57d92be56177fde0e3cdd58bc05ba28f3/source/plugins/core/metadata.yml#L91-L93

**Reply by ThuWangzw** [reply]
It works! Thank you 😄
